packages/format-library/src/image/index.js

```diff
@@ -87,11 +87,11 @@ export const image = {
 
 			return (
 				<MediaUploadCheck>
-					<RichTextInserterItem
-						name={ name }
+					<RichTextToolbarButton
```

Comment:
Does it mean we no longer want to keep `RichTextInserterItem` component?

Reply:
Yes, let's remove it. It hasn't been documented.